asciidoc/volume3/biceps-extension-provisions/extension-report-sequence.adoc

```diff
@@ -0,0 +1,75 @@
+.<<vol2_clause_appendix_mdpws_dev_29, {var_label_dev_29_message_metricreport}>> with report sequence extension.
+[#vol3_example_extension_report_sequence]
+====
+[source,xml]
+----
+include::../../listings/vol3-clause-biceps-content-example-report-sequence.xml[]
+----
+====
+
+
+[sdpi_level=+1]
+====== Requirements
+
+.R0800
+[sdpi_requirement#r0800,sdpi_req_level=shall]
+****
+The <<vol1_spec_sdpi_p_actor_somds_provider>> shall set the `sdpi:ReportSequence/@MessageNumber` to 1 for the first notification message sent in a new publish-subscribe session.
+
+.Notes
+[%collapsible]
+====
+NOTE: A new publish-subscribe session begins when the <<vol1_spec_sdpi_p_actor_somds_provider>> receives a Subscribe message.
+
+NOTE: A <<vol1_spec_sdpi_p_actor_somds_provider>> may send the first report before sending the SubscribeResponse message.
+
+NOTE: The `sdpi:ReportSequence/@MessageNumber` is scoped to the publish-subscribe session. Different subscriptions may share the same `sdpi:ReportSequence/@MessageNumber`.
+====
+****
+
+.R0801
+[sdpi_requirement#r0801,sdpi_req_level=shall]
+****
+The <<vol1_spec_sdpi_p_actor_somds_provider>> shall increment the `sdpi:ReportSequence/@MessageNumber` by exactly 1 only when it receives confirmation of successful delivery of the notification message.
+
+.Notes
+[%collapsible]
+====
+NOTE: A <<vol1_spec_sdpi_p_actor_somds_provider>> only increments the `sdpi:ReportSequence/@MessageNumber` for successfully delivered notification messages.
+
+NOTE: Successful delivery for <<vol1_spec_sdpi_p_actor_somds_participant>>s using <<ref_w3c_soap_over_http_v1_2, SOAP over HTTP transport>> is indicated by an HTTP status code between 200 and 299.
+
+NOTE: A response may be generated by a server layer before the message reaches a <<vol1_spec_sdpi_p_actor_somds_consumer>>. HTTP response 503 (service unavailable) is an example of a status code that may be generated by a server layer. The <<vol1_spec_sdpi_p_actor_somds_provider>> does not change the `sdpi:ReportSequence/@MessageNumber` when it receives a fault status response. 
+
+NOTE: An absent response could mean either the notification or the notification response was not delivered due to a network fault.
+
+====
+****
+
+.R0802
+[sdpi_requirement#r0802,sdpi_req_level=shall]
+****
+Two reports sent with the same `sdpi:ReportSequence/@MessageNumber` shall be functionally identical unless the <<vol1_spec_sdpi_p_actor_somds_provider>> receives a fault response to a notification message from the recipient.
+
+.Notes
+[%collapsible]
+====
+NOTE: A <<vol1_spec_sdpi_p_actor_somds_provider>> may change the content of a report to resolve a fault when it receives a fault response. 
+
+NOTE: A timeout (no response or partial response) should not be considered a fault response from the recipient because message may have been received by the recipient's response may not have been delivered. A <<vol1_spec_sdpi_p_actor_somds_provider>> doesn't change the message content if if receives no response. 
```

Comment:
Can you clarify what you mean by this note? I am not sure I really got the gist of it.